boot.php

```diff
@@ -1584,3 +1585,41 @@ function infinite_scroll_data($module)
 		return $arr;
 	}
 }
+
+/**
+ * @brief checks if there is a autoconfig.php file in the config folder and setup friendica in case of
+ *
+ * In an automatic setup, autoconfig.php could be created under the "config/" folder with default values
+ * for the mysql instance. At the end, this file should get deleted.
+ *
+ * @return true if the autoconfig was succesfully, otherwise false
+ */
+function autoconfig()
```

Comment:
Is "autoinstall" maybe a better name for this function? It would fit to `mod/install.php` and `Core/Install.php`

Reply:
You're right, it does. It uses an autoconfig file but it is performing the install.